recipes/Kili_fine_tuning_tutorial.ipynb

```diff
@@ -0,0 +1,2081 @@
+{
```

Comment:
<p>I think that here you should explain that each line of this file is an input text and its class, separated by an "|"</p><p> </p><p>Also, I would change the code like this instead:</p><p> </p><pre class="ql-syntax" spellcheck="false">
with open(...) as f:
	lines = f.readlines()
 
lines = [line.split("|") for line in lines]
content_array = [line[0] for line in lines]
categories_array = [line[1] for line in lines]
external_id_array = [f"text_{i}" for i, _ in enumerate(content_array)]
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1259/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>1: addressed in NB</p><p>2. Will address this in version 2.0 of the NB</p>